tests/integration/utils.py

```diff
@@ -199,10 +210,18 @@ def fetch_deps_and_check_output(
         expected_files_checksums = load_json(
             os.path.join(test_data_dir, test_case, "fetch_deps_sha256sums.json")
         )
+
         log.info("Compare checksums of fetched deps files")
-        assert (
-            files_checksums == expected_files_checksums
-        ), f"Expected files checksusms:/n{files_checksums}"
+        for dep in expected_files_checksums.keys():
+            actual = files_checksums[dep]
+            expected = expected_files_checksums[dep]
+
+            # black demands that we make the assertion statement a tuple here
```

Comment:
Let me know if there's a better way to fix this. I got tired of trying, I'll be back to it tomorrow.

Reply:
Coming at this from a slightly different direction. We want to able to auto-generate test data, which IMO means that we want to keep `assert files_checksums == expected_files_checksums`

The expected data should already be generated appropriately. For example
```python
if "external-gitcommit" in tarball_path.name:
    expected_checksum = f"gitcommit:{get_commit_in_repo(tarball_path)}"
else:
    expected_checksum = f"sha256:{sha256sum(tarball_path)}"
```

And maybe we *should* check the commit, otherwise the test is a bit weak